### PR TITLE
updating the interface of processEclipseFormat

### PR DESCRIPTION
**Pull request by GitPaean**
due to the upstream change.

### CODE REVIEW

**Review by bska — APPROVED** [review]
Looks good.  Just trivial changes following an API update.  I'll merge this once the upstream changes are merged.

**Review by blattms — APPROVED** [review]
Approved if jenkins says yes

**Pull request merged**
